db/migrate/20200406183029_add_cols_to_pruebas.rb

```diff
@@ -0,0 +1,7 @@
+class AddColsToPruebas < ActiveRecord::Migration[6.0]
+  def change
+    add_column :pruebas, :nombre, :string
+    add_column :pruebas, :edad, :int
+    add_column :pruebas, :fechaNac, :string
```

Comment:
Change this columns name for `fecha_nacimiento`. Also, since this is a date, your column type should be date, not a string.